<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditItemDialog</class>
 <widget class="QDialog" name="EditItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>162</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>214</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="okCancelLayout">
    <item>
     <widget class="QDialogButtonBox" name="okCancelButton">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>148</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="itemEntryLayout">
    <item>
     <layout class="QHBoxLayout" name="itemNameLayout">
      <item>
       <widget class="QLineEdit" name="itemNameEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="itemNameLabel">
        <property name="text">
         <string>Item Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="itemCostLayout">
      <item>
       <widget class="QDoubleSpinBox" name="itemCostSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="itemCostLabel">
        <property name="text">
         <string>Item Cost</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="itemStockLayout">
      <item>
       <widget class="QSpinBox" name="itemStockSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="itemStockLabel">
        <property name="text">
         <string>Stock Level</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="reorderCheckBox">
      <property name="text">
       <string>Reorder?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButton</sender>
   <signal>accepted()</signal>
   <receiver>EditItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButton</sender>
   <signal>rejected()</signal>
   <receiver>EditItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
